operations/app/src/modules/network/vpn.tf

```diff
@@ -1,121 +0,0 @@
-/* DNS server for VPN */
-resource "azurerm_network_profile" "vpn_dns_network_profile" {
```

Comment:
This is the crux of this PR, removing the VPN Gateway in the Palo Alto VNET. Note: The VPN Gateway still exists in `old-vnet.tf` so no changes apply to the current VNETs.